.gitmodules

```diff
@@ -10,4 +10,7 @@
 	branch = master
 [submodule "dsymbol"]
 	path = dsymbol
-	url = https://github.com/dlang-community/dsymbol.git
\ No newline at end of file
+	url = https://github.com/dlang-community/dsymbol.git
+[submodule "stdx-allocator"]
+	path = stdx-allocator
+	url = https://github.com/dlang-community/stdx-allocator
```

Comment:
url misses the `.git` i think

Reply:
no b/c dscanner doesn't require it either

Reply:
Oh the problem is just that the make file is not updated

Reply:
warning: done in your fork

Reply:
FWIW `.git` isn't required. GitHub is smart enough for this

Reply:
Thanks a lot for updating the Makefiles!